conmon/conmon.c

```diff
@@ -545,6 +623,33 @@ int main(int argc, char *argv[])
 		}
 	}
 
+	/* Setup OOM notification for container process */
+	memory_cgroup_path = process_cgroup_subsystem_path(cpid, "memory");
+	if (!memory_cgroup_path) {
+		nexit("Failed to get memory cgroup path");
+	}
+
+	bool oom_handling_enabled = true;
+	char memory_cgroup_file_path[PATH_MAX];
+	snprintf(memory_cgroup_file_path, PATH_MAX, "%s/cgroup.event_control", memory_cgroup_path);
+	if ((cfd = open(memory_cgroup_file_path, O_WRONLY)) == -1) {
+		nwarn("open cgroup.event_control");
+		oom_handling_enabled = false;
+	}
+
+	if (oom_handling_enabled) {
+		snprintf(memory_cgroup_file_path, PATH_MAX, "%s/memory.oom_control", memory_cgroup_path);
+		if ((ofd = open(memory_cgroup_file_path, O_RDONLY)) == -1)
+			pexit("open memory.oom_control");
```

Comment:
I'm not able to find the pexit function quickly, but if it's just passing this string along, I'd add "Unable to" or "Failure opening" or some such.  Ditto the other pexits through out.